docs/source/user-guide/integrations/hugging_face.rst

```diff
@@ -0,0 +1,233 @@
+HuggingFace Transformers
+========================
+
+This tutorial demonstrates how deepchecks.vision can be used on a Hugging Face transformer model. We will use deepchecks
+to compare the performance of the `DETR ResNet <https://huggingface.co/facebook/detr-resnet-50>`__ transformers model
+against the widely used `YOLOv5s <https://arxiv.org/abs/1804.02767>`__ model on the `COCO <https://cocodataset.org/>`__
+dataset.
+
+Load COCO and YOLOv5s
+------------------------
+
+We'll start by loading from ``deepchecks.vision.datasets.detection.coco`` a sample of the COCO dataset (coco 128) and
+the YOLO model, both downloaded from `ultralytics <https://github.com/ultralytics/yolov5>`__ repository.
+
+.. code-block:: python
```

Comment:
why not using the .. literalinclude? This way you can let the user download the full code

Reply:
Not sure I follow. We're downloading both using deepchecks code, but the download is from an ultralytics repo. What should be included?